src/components/ui/AGENTS.md

```
@@ -0,0 +1,107 @@
+# UI COMPONENTS (shadcn/ui)
+
+**Vendor Code:** Third-party UI primitives from shadcn/ui project
+
+## OVERVIEW
+shadcn/ui base components (Radix UI primitives + Tailwind styling). These are **vendor-supplied patterns**, not application code.
+
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)